pyproject.toml

```diff
@@ -13,6 +13,13 @@ seaborn = "^0.10.0"
 matplotlib = "^3.1.3"
 pytest-flask = "^0.15.1"
 pytest = "^5.3.5"
+Flask-SQLAlchemy = "^2.4.1"
+flask-marshmallow = "^0.11.0"
+marshmallow-sqlalchemy = "^0.22.3"
+marshmallow = "^3.5.0"
+mock = "^4.0.1"
+aiosql = "^3.0.0"
+flask_sqlalchemy = "^2.4.1"
```

Comment:
Dependencies that should be added in the database stage.
You should remove them here.